docs/data/data-grid/rows/RowOrderingGrid.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+import {
+  DataGridPro,
+  GridRowModel,
+  GridRowOrderChangeParams,
+} from '@mui/x-data-grid-pro';
+import { useDemoData } from '@mui/x-data-grid-generator';
+
+function updateRowPossition(
+  initialIndex: number,
+  newIndex: number,
+  rows: Array<GridRowModel>,
+): Promise<any> {
+  return new Promise((resolve) => {
+    setTimeout(() => {
+      const rowsClone = [...rows];
+      const row = rowsClone.splice(initialIndex, 1)[0];
+      rowsClone.splice(newIndex, 0, row);
+      resolve(rowsClone);
+    }, Math.random() * 500 + 100); // simulate network latency
+  });
+}
+
+export default function RowOrderingGrid() {
+  const { data } = useDemoData({
+    dataSet: 'Commodity',
+    rowLength: 20,
+    maxColumns: 20,
+  });
+
+  const [rows, setRows] = React.useState(data.rows);
+  const [loading, setLoading] = React.useState(false);
+
+  React.useEffect(() => {
+    setRows(data.rows);
```

Comment:
> It would be nice to also set __reorder__. The ID is a hidden field in this dataset.

I didn't get that why do we need to set the special column?
For the other part of the feedback - done.

Reply:
I was meaning this

![msedge_h8kpx80D49](https://user-images.githubusercontent.com/42154031/164464711-543375a3-3048-44bc-ba08-ae6ae45f8a3c.gif)

It shows the ID as placeholder, but the ID is not visible.

Reply:
done

Reply:
I still see the ID.

Reply:
in this demo https://deploy-preview-4034--material-ui-x.netlify.app/x/react-data-grid/rows/#row-reorder - yes. I didn't update it but I've added a section explaining how to customize it.